packages/backend/src/routers/oauthRouter.ts

```diff
@@ -433,4 +421,12 @@
     oauthProtectedResourceHandler,
 );
 
+// MCP server protected resource discovery endpoint
+// This endpoint is used to discover the protected resource for MCP
+// Required by some MCP clients
+oauthRouter.get(
+    '/.well-known/oauth-protected-resource/api/v1/mcp',
```

Comment:
by adding this endpoint on the router, it will be available under

`/api/v1/oauth/.well-known/oauth-protected-resource/api/v1/mcp`

You probably only need the one in App.tsx